sys/include/div.h

```diff
@@ -0,0 +1,16 @@
+#ifndef DIV_H
+#define DIV_H
+
+#include <stdint.h>
+
+static inline uint64_t div_u64_by_1000000(uint64_t val)
```

Comment:
hm I like this approach. If it's really more effective then I will adopt the in RPL.